product_docs/docs/bdr/4/release_notes/bdr4.1.1_rel_notes.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: "BDR 4.1.1"
+---
+
+This is a maintenance release of BDR 4 which includes new features as well
+as fixes for issues identified in previous versions.
+
+
+| Type | Category | Description |
+| ---- | -------- | ----------- |
+| Feature | User Experience | Add generic function bdr.is_node_connected returns true if the walsender for a given peer is active.
+| Feature | User Experience | Add generic function bdr.is_node_ready returns boolean if the lag is under a specific span.
+| Feature | User Experience | Add support for a `--link` argument to bdr_pg_upgrade for using hard-links.
+| Bug fix | Reliability and operability | Correct Raft based switching to Local Mode for CAMO pairs of nodes (RT78928)
```

Comment:
So couple things:
 - --link is not feature but bugfix
 - could we mention BDR-2270?